CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+# Contributing to OSRD
+
+## Where to start ?
+
+All contributions, bug reports, bug fixes, documentation improvements, enhancements, and ideas are welcome.
+
+If you are brand new to OSRD or open-source development, we recommend chatting with us on [libera chat](https://web.libera.chat/#osrd) or going through the [GitHub issues](https://github.com/DGEXSolutions/osrd/issues) tab to find issues that interest you. There are number of issues listed under `good first issue` where you could start out.
+
+## Contribute code
+
+Commit style is detailed in the [osrd.fr/docs](https://osrd.fr/en/docs/guides/contribute/code/#git-commit-style).
+
+### Pull request / code review
+
+- Every PR, good or bad, is an act of generosity.
+  Opening with a positive comment will help the author feel rewarded, and your subsequent remarks may be heard more clearly.
+  You may feel good also.
+- Add a description to PRs to explain what they do and why.
+- Each PR must be linked to at least one github issue to simplify tracking.
+- Pay attention to all the comments made by reviewers and either answer/challenge them or address them before asking for another review.
+- Assign one maintainer and one developer using github assignment when opening the PR to get reviews.
+- Help the reviewer by following advice given in [mtlynch article](https://mtlynch.io/code-review-love/).
+- Review code following advice given in [osrd.fr/docs](https://osrd.fr/en/docs/guides/contribute/code-reviews/) and [mtlynch article](https://mtlynch.io/human-code-reviews-1/).
+- Make commits of logical and atomic units.
+- If possible, make PR of logical and atomic units too (avoid mixing refactoring, new features and bug fix at the same time).
+- Keep your branch up-to-date :
+```
+git checkout <your_branch>
+git fetch
+git rebase origin/dev
+```
+
+### Bugs
+
+The way support is delivered greatly impacts how users perceive the product and the team.
+Handling the incoming support requests efficiently is thus a top priority.
+
+- Bug must have a correct description and the bug's issue template must be filled carefully.
+- Bug must be tagged with:
+  - `kind:bug`
+  - one or several `area:<affected_area>` if possible, if the affected area is not known leave it blank it will be added later by another team member`.
+  - one `severity:<bug_severity>` if possible, if severity is not known leave it blank it will be added later by another team member
```

Comment:
It would be useful to explain here the meaning of each severity level:
- minor: user can still use the feature
- major: user sometimes can't use the feature
- critical: user can't use the feature